src/components/shared/Entity/WrapperWithHeader/index.tsx

```diff
@@ -53,7 +53,7 @@ function WrapperWithHeader({
             expanded={expanded}
             onChange={handlers.change}
             sx={{
-                mt: 2,
+                my: 2,
```

Comment:
When a section of holistic create/edit form is unable to be rendered and an error alert takes its place, this margin ensures adequate space exists between the alert and the form section above it.

Reply:
Is there a reason we don't add the space to the error itself?